config/config.go

```diff
@@ -182,32 +182,32 @@ func (data *ExternalCache) validate(errs []error) []error {
 	}
 
 	if data.Scheme != "" && data.Scheme != "http" && data.Scheme != "https" {
-		return append(errs, errors.New("External cache Scheme must be http or https if specified"))
+		return append(errs, errors.New("external cache Scheme must be http or https if specified"))
```

Comment:
The case changes are potentially breaking, please revert.

Reply:
done